src/index.ts

```diff
@@ -91,6 +95,8 @@ export {
   RealtimeComponentState,
   RealtimeComponentEvent,
   CommentEvent,
+  ComponentLifeCycleEvent,
+  WhoIsOnlineEvent,
```

Comment:
```suggestion
  ComponentLifeCycleEvent,
  WhoIsOnlineEvent,
  CommentEvent,
```

Reply:
`CommentEvent` is right above `ComponentLifeCycleEvent` in the original code